# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/xhr/send-authentication-competing-names-passwords.htm.ini
# Commit: ef6d81f0b298
# Full Hash: ef6d81f0b2989624a9918ec5d790dc2745db20ef
# Author: Valentin Gosu <valentin.gosu@gmail.com>
# Date: 2018-11-15 09:52:43
# Regressor Bug: 1495313
# File Overlap Count: 1
# Description:
#   Bug 1495313 - Allow password with empty username in URLs r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D11254
# ==============================================================================

diff -r 06b12fd41ff1 -r ef6d81f0b298 testing/web-platform/meta/xhr/send-authentication-competing-names-passwords.htm.ini
--- a/testing/web-platform/meta/xhr/send-authentication-competing-names-passwords.htm.ini	Wed Nov 14 19:37:42 2018 +0000
+++ b/testing/web-platform/meta/xhr/send-authentication-competing-names-passwords.htm.ini	Wed Nov 14 19:10:42 2018 +0000
@@ -4,10 +4,3 @@
 
   [XMLHttpRequest: send() - "Basic" authenticated requests with competing user name/password options user name and password in URL userinfo, only user name in open() call: user name in open() wins]
     expected: FAIL
-
-  [XMLHttpRequest user/pass options: pass in URL, user in open()]
-    expected: FAIL
-
-  [XMLHttpRequest user/pass options: pass in URL, user/pass in open()]
-    expected: FAIL
-
